docs/design-decisions/README.md

```diff
@@ -5,26 +5,64 @@
 ## Outstanding Questions
 
 - Q: How does the switch _know_ to send a callback to the PISP after a sucessful transfer?
-  - when DFSPA issues `POST /transfer`, that's between DFSPA  + DFSPB
-  - It should have a `transferId` that is related to the `thirdPartyRequest`
-    - how does the PISP or Switch know which transaction this `transferId` is related?
-    - tx id will be set in the `POST /authorizations` (when the authorization is created by the DFSP)
-    - how do we get from a `transactionId` to a `transferId`?
-      - ILP packet is in transfer request, but encoded... so need to figure this piece out
-    - For now, assume that the `Transaction` object will contain the `transferId`
-  > Note: this is being tracked in [#270](https://app.zenhub.com/workspaces/pisp-5e8457b05580fb04a7fd4878/issues/mojaloop/mojaloop/270)
+    - Refer to [#42](https://github.com/mojaloop/pisp/issues/42)
```

Comment:
Just a little curious over here, what are the design considerations behind storing a subscription object for every transaction? Why don't we just let the PISP subscribe to every event with `FSIOP-Destination` equals to the unique identifier of the PISP?

Reply:
If the `FSIOP-Destination` is equal to the the pisp's ID, then by the nature of how the mojaloop requests are routed, they indeed will be delivered to the PISP.

The challenge is how to get requests with a `FSIOP-Destination` header _not equal_ to the PISP's ID, but that a PISP needs to be informed about. Such as `PUT /transfers/{id}` from the PayeeDFSP.

Reply:
Ohh I see, right, got it, thank you :)